lib/seed.dart

```diff
@@ -18,6 +19,25 @@ class _SeedState extends State<Seed> {
   bool checked = false;
   bool visibility = false;
 
+  // initialise the phrase with empty words - in order for the widget to not throw
+  // an error while waiting for the rust api. Seems to be the easiest way of handling
+  // uninitialised state, as the words will be ready immediately after the widget is
+  // initialised and the words are not visible at first.
+  List<String> phrase = ["", "", "", "", "", "", "", "", "", "", "", ""];
+
+  @override
+  void initState() {
+    _callGetSeedPhrase();
```

Comment:
❓ Can it happen that this call never finishes or can we be sure that the seed is available through the API?

Reply:
> Can it happen that this call never finishes
It could, but it wouldn't block the screen.

> can we be sure that the seed is available through the API?
No we can't, e.g. if the wallet is not initialised this call will fail. (that's the reason for the init wallet in the integration tests).

Error handling should be there, but flutter complains about simply printing the error so I opened the logging PR to be able to log errors properly. I will adapt here once https://github.com/itchysats/10101/pull/104 is merged.

Reply:
Added error handling https://github.com/itchysats/10101/pull/102/commits/451f23039bf77b019f914b33f5a358c2904dfda0